src/components/BucketlistView.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import ROOT_URL from '../constants';
+import axios from 'axios';
+import AddItemForm from '../forms/AddItemForm';
+import ItemCard from './ItemCard';
+import InNavAppBar from '../navigation/InNavAppBar';
+
+class BucketlistView extends Component {
+  constructor() {
+    super();
+    this.state = {
+      bucket_list_id: null,
+      items: []
+    }
+  }
+
+  componentWillMount() {
+    this.getItems();
+  }
+
+  getItems() {
+    var items = {};
+    axios({
+      method: 'get',
+      url: ROOT_URL + '/api/v1/bucketlists/' + this.props.match.params.id + '/items',
+      headers: {
+        "Authorization": `JWT ${window.sessionStorage.accessToken}`,
+      }
+    }).then(response => {
+      items = response.data.items;
+      this.setState({
+        items: items,
+      })
+    }).catch(error => {
+      return error;
+    });
+  }
+
+  getBucketlist() {
+    axios({
+      method: 'get',
+      url: ROOT_URL + '/api/v1/bucketlists/'+this.props.location.id,
```

Comment:
Consider abstracting the http calls outside for additional semantic structure. No?